### PR TITLE
LPS-178858 | Provide meaningful empty states (DataSets)

### PR DESCRIPTION
**Pull request by andreluis2022**
**Description**:
Product QA | Functional Automation - LPS-178858 Provide meaningful empty states

**JIRA Ticket:**
[LPS-178858](https://liferay.atlassian.net/browse/LPS-178858) Story

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:relevant

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete can you review these tests ?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase [hunk 1]
```diff
@@ -362,6 +422,30 @@ definition {
 		}
 	}
 
+	@description = "LPS-178858 Confirm that the modal data set view is displayed"
+	@priority = 5
+	test CanAssretTheDataSetViewCanBeCreate {
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase:427]: Fix the name of the test, it should be CanAssertTheDataSetViewCanBeCreate

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase (hunk 1 above)
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase]: @CarlosBrichete what do you think?

**Comment by andreluis2022** [comment]
ci:test:relevant

**Comment by andreluis2022** [comment]
ci:help

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:forward

**Comment by CarlosBrichete** [comment]
hey @andreluis2022 you have to fix the source format first

**Comment by andreluis2022** [comment]
ci:forward

**Pull request closed without merging**